20HW/1_2_task/src/data/types/product.types.ts

```diff
@@ -0,0 +1,20 @@
+export interface IProduct {
+  name: string;
+  manufacturer: MANUFACTURERS;
+  price: number;
+  amount: number;
+  notes?: string;
+}
+
+export enum MANUFACTURERS {
+  APPLE = "Apple",
+  SAMSUNG = "Samsung",
+  GOOGLE = "Google",
+  MICROSOFT = "Microsoft",
+  SONY = "Sony",
+  XIAOMI = "Xiaomi",
+  AMAZON = "Amazon",
+  TESLA = "Tesla",
+}
+
+export type keys = keyof IProduct;
```

Comment:
`keys` - ключи от чего ? 
```suggestion
export type TProductKeys = keyof IProduct;
```

Reply:
Это остатки уже прошедшей идеи. :)